insert_data_to_vector_db_weaviate/README.md

```diff
@@ -0,0 +1,44 @@
+# Insert a set of embedding vectors to Weaviate
```

Comment:
Readme file doesn't mention anything about data/ folder.

Reply:
L21 of README.md instructs the user to `Create a new Google sheet in your Google account and import `data/data.csv` to the sheet from columns **A:C**.`